src/webroot/css/cake-core.min.css

```diff
@@ -0,0 +1 @@
+.cake-core-hidden{display:none}
```

Comment:
No min files please. It complicates maintenance, and people who care about such things can use tools like assetcompress/grunt/gulp/etc.